core/src/main/java/org/apache/iceberg/BaseTransaction.java

```diff
@@ -316,20 +317,13 @@ private void commitCreateTransaction() {
 
     } catch (CommitStateUnknownException e) {
       throw e;
-
```

Comment:
Nit: unnecessary whitespace change. I also like the whitespace here because it maintains separation.

Reply:
Sure, I added back the newline